content/events/20-21/061_dnd.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Zeus Doet Dungeons & Dragons!'
+description: Zeus Doet Dungeons & Dragons (en andere tabletop RPG's)-avond
+created_at: 18-11-2020
+time: '28-11-2020 19:00'
+location: 'Bij je thuis'
+banner: https://zeus.ugent.be/zeuswpi/XqCSUGHo.jpg
+facebook: https://www.facebook.com/events/1522197801317546/
+---
```

Comment:
I'd suggest setting `header_text_background` to `true`, for better readability:

![image](https://user-images.githubusercontent.com/1756811/99594921-25ed4d80-29f4-11eb-8cd0-53d82d4b1f16.png)